### ISSUE TITLE
TimePicker unit tests AccessibilityAttributeVerification and Redlines2 need to be updated

### ISSUE THREAD

**Issue by TheWrathOfConst** [winjs/winjs#1148]
These are failing after commit f6b268b8277837c8a3dc59ad37f6709785d59126 

Redlines2 is failing because the background color changed from what it expected:
areEqual - testing subElement=win-timepicker-hour win-dropdown win-order0, style=color (expected: rgb(33, 33, 33), actual: rgb(255, 255, 255))

AccessibilityAttributeVerification is failing because we added a new CSS class to the TimePicker elements, and this test tries to do an exact string comparison against TimePicker element class names.
